index.html

```diff
@@ -0,0 +1,292 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>My personal Portfolio</title>
+  </head>
+  <body>
+    <!-- Top-navigation bar -->
+    <nav id="top-nav">
+      <div class="item-nav">
+        <div class="item-1">
+          <div class="item-name-1">Portoflio</div>
+        </div>
+
+        <div class="item-2">
+          <div class="item-name-2">About</div>
+        </div>
+
+        <div class="item-1">
+          <div class="item-name-3">Contact</div>
+        </div>
+      </div>
+
+      <div class="name-top">
+        <div class="name">Patrick Maina</div>
+      </div>
+
+      <div class="mail-button">
+        <div class="mail-icon">
+          <img src="Union.svg" alt="Mail box" />
+        </div>
+      </div>
+    </nav>
+    <!-- Bottom-navigation-bar -->
+    <nav id="bottom-nav">
+      <div class="bottom-border"></div>
+      <div class="social-media">
+        <ul class="tags">
+          <li>
+            <img src="GitHub.png" alt="GitHub icon" />
+          </li>
+
+          <li>
+            <img src="LinkedIn.png" alt="LinkedIn icon" />
+          </li>
+
+          <li>
+            <img src="Angelist.png" alt="Angelist icon" />
+          </li>
+
+          <li>
+            <img src="Twitter.png" alt="Twitter icon" />
+          </li>
+
+          <li>
```

Comment:
- [ ] Your background is missing this small content kindly implement it.
### Figma design
![figma](https://user-images.githubusercontent.com/81727988/171833303-49ad2d03-bf2a-4baa-b322-bf5d6aab313b.png)